### ISSUE TITLE
Flag acceptAutoSigned not working for image download

### ISSUE THREAD

**Issue by ermedeiros** [usharesoft/hammr#58]
Even after setting the flag "acceptAutoSigned" as true in the credentials file, Hammr fails to download an image.
Version: 0.2.5.9

Command:
`hammr image download --id 1095 --file image.tar.gz`

Output:

```
INFO: no username nor password provided on command line, trying credentials file
INFO: Using credentials file: /home/elias/.hammr/credentials.json
INFO: Using url https://10.1.2.151/api
Searching image with id [1095] ...
ERROR: downloading https://10.1.2.151/api/users/elias/appliances/747/images/1095/downloads/uss-dev-env_1.0rev1_targz.tar.gz?downloadId=e662d401bd7313b9db1fa9d50ce3b85f: [Errno socket error] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)

```

Work arround: use http instead of https.

**Comment by segalaj** [comment]
work done on ussclicore: https://github.com/usharesoft/ussclicore/pull/18
need to be merged before

**Comment by DimitriSCOLE** [comment]
It seems that the issue is always there due to an incompatibility with python 2.6

**Comment by segalaj** [comment]
@DimitriSCOLE this issue is still open. If you found another problem, please create a new issue. Otherwise please add more information.

**Comment by DimitriSCOLE** [comment]
The informations are written above...

**Comment by AliceAlbano** [comment]
The work is done in PR [201](https://github.com/usharesoft/hammr/pull/201). I added in download method an optional argument (introduced by the modification in ussclicore discussed above). Until now, this argument had a default value, now we use the value given to hammr.
